.github/workflows/test.yml

```diff
@@ -96,10 +96,9 @@ jobs:
         with:
           required-ros-distributions: ${{ matrix.ros_distribution }}
       - uses: ./
-        id: test_single_package
-        name: "Test single package, default options"
+        id: test_all_packages_in_repo
```

Comment:
you could add a `test -d …` check below for `test_osrf_testing_tools_cpp` since it should now get built